tf2_web_republisher/include/tf_web_republisher/tf2_web_republisher.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "sstream"
```

Comment:
Add explicit includes for `string` and `memory` (for shared_ptr). Also, system headers are typically included using `<>` brackets
```suggestion
#include <memory>
#include <sstream>
#include <string>
```